openapi.yaml

```diff
@@ -8578,3 +8626,142 @@ components:
       - title
       - total_stats
       type: object
+    FilterDeploymentsRequest:
+      additionalProperties: false
+      description: Filters to select the deployments in `/filter/deployments`.
+      example:
+        account: 1
+        date_from: 2021-06-23
+        date_to: 2021-12-24
+        in:
+        - github.com/athenianco/athenian-api
+        - github.com/athenianco/athenian-webapp
+        - github.com/athenianco/metadata
+        environments:
+        - production
+        - staging
+        conclusions:
+        - FAILURE
+        with_labels:
+          user: github.com/lwsanty
+          rollback: null
```

Comment:
Are those 2 labels present by default in the response and with a null value if the info is not given by the user?

Reply:
I've given a long good description how it works in the description of `with_labels` :wink:

Reply:
No labels - not present. The same way like all the rest of our endpoints work, no surprises.

Reply:
No my question is more about events/deployment then. How a deployment can have the label `rollback` with the value null?

Reply:
The answer is https://github.com/athenianco/api-spec/pull/13/files#diff-d910ba2ef878f7db0223a966b81c8b3f3b65027bb39e4431bb05140171eece39R8691

Reply:
The being said, labels are free-form, the user can send any JSON value for a label including `null`.

Reply:
Ok I finally get it. The example for rollback confused me as a user would want to filter deployments where rollback is either true or false but checking the existence of the label was weird to me.

Reply:
I just suspect that 95% of possible filters by label will not care about the specific value. But let's see. There are two ways to mark a rollback in the end:
```
rollback: true || false
```
and
```
rollback: "whatever" // is a rollback
// missing "rollback" label if not a rollback
```

Reply:
Ok got it!